files/create_backup.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Configure the GraphDB backup cron job
+
+cat <<-EOF > /usr/bin/graphdb_backup
+#!/bin/bash
+
+set -euxo pipefail
+
+GRAPHDB_ADMIN_PASSWORD="\$(aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/admin_password" --with-decryption | jq -r .Parameter.Value)"
+NODE_STATE="\$(curl --silent --fail --user "admin:\$GRAPHDB_ADMIN_PASSWORD" localhost:7201/rest/cluster/node/status | jq -r .nodeState)"
```

Comment:
By default GraphDB nodes are started on port 7200.
@yaskoo perhaps we should move the proxies to port 7201?

Reply:
Not the biggest fan, but the script could accept a parameter, or look for an env var for it with a default value of `7200`?